CHANGELOG.md

```diff
@@ -1,81 +0,0 @@
-## Changelog
```

Comment:
@dangilk I agree, we should remove the entire integration guide folder from the repo and probably from your build process. This way we can keep all the docs in one place and keep this repo in parity with all others.